tests/test_peak_detection.py

```diff
@@ -53,6 +55,54 @@ def test_peak_detection__analyzes_data_correctly_when_twitches_point_down():
         assert non_flipped_valleys[idx] == flipped_valleys[idx], idx
 
 
+@pytest.mark.parametrize(
+    "input_tuple,test_description",
+    [
+        ((1, 1), "Test regular"),
+        ((1,), "Test oneInput tuple"),
+    ],
+)
+def test_peak_detection_input__prominence_factors(input_tuple, test_description, mocker):
+    filtered_magnetic_signal = np.zeros(shape=(2, 10))
+    filtered_magnetic_signal[0, 1] = 10
+    filtered_magnetic_signal[0, 0] = 5
+
+    m = mocker.spy(peak_detection, "peak_detector")
+    assert (
+        peak_detection.peak_detector(
+            prominence_factors=input_tuple,
+            filtered_magnetic_signal=filtered_magnetic_signal,
+            start_time=0,
+            end_time=10,
+        )
+        != None
```

Comment:
Few things here
- When comparing to `None`, `False,` or `True`, always use `is` or `is not`. Using `==` or `!=` can lead to subtle bugs
- We don't actually care about the output of `peak_detector` here, all we care about is that the scipy `find_peaks` function is called with the correct values for prominence. What this means is that we should actually be mocking that function instead and making an assertion on the args it's called with
- We probably also want to test just passing a single int value that is not in a tuple